content/docs/guides/wiki-contribution-guide.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Wiki Contribution Guide"
+description: "Guides specific for this website internals"
+summary: ""
+date: 2023-09-07T16:04:48+02:00
+lastmod: 2023-09-07T16:04:48+02:00
+draft: true
+weight: 810
+toc: true
+version: 1.0
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Fonts
+
+[**CMU Sans Serif**](https://online-fonts.com/fonts/cmu-sans-serif) This font is used in the diagrams and figures. Make sure to have this font installed on your machine to render the figures correctly.
+
+## Colors
+
+The color palette used in the Oakestra documentation is as follows:
+
+![OakBlue](wiki-contribution-guide/oak-blue.png) Oakestra Blue Background: R:73 B:114 G:134 #497286
+
+![OakGreen](wiki-contribution-guide/oak-green.png) Oakestra Tree Green: R:125 B:195 G:132 #7DC384
+
+## Drawio master DB
+This is the reference drawio diagram for the Oakestra project.
+https://drive.google.com/file/d/1zfgXwXPI2Yb8G2N-U8Gd0xPaiQIe9R-2/view?usp=sharing
+
+New Pictures should be added to this DB.
+
+## Drawio files in the docs
+Some pictures that are not yet present in the Drawio master DB have the corrspondign .drawio file in the docs. 
```

Comment:
```suggestion
Some pictures that are not yet present in the Drawio master DB have the corresponding .drawio file in the docs. 
```

What do you mean by "not yet" - should all pictures be in that master file? - Should we make a policy or rule of thumb for that? - e.g. if a picture is most likely only used in a single place, than do not put it into the master file otherwise do